autoload/maktaba/extension.vim

```diff
@@ -54,6 +56,8 @@ function! maktaba#extension#GetInternalRegistry(plugin) abort
     return s:registries[a:plugin]
   endif
 
+  call s:maktaba.logger.Info(
+      \ 'New extension registry for plugin ''' . a:plugin . '''')
```

Comment:
Something like `Info('New extension registry for plugin "%s"', a:plugin)` instead?